docker/Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN apt-get update && \
 RUN mkdir -p /opt/source-code/feagi-core/ && \
     git clone $REPO /opt/source-code/feagi-core
 WORKDIR /opt/source-code/feagi-core/
-RUN git checkout feature-dockerize-lidar
+RUN git checkout develop
```

Comment:
I don't think this line is necessary, but its presence isn't breaking anything. The default branch is `develop` after cloning (correct me if I'm wrong...).

Reply:
good catch! I think it would be best to have that line removed so it uses the default at all time. I will fix and will send it back.